llama-rs/src/lib.rs

```diff
@@ -0,0 +1,7 @@
+mod ggml;
+mod llama;
+
+pub use llama::{
+    GptVocab as Vocab, InferenceParams, LlamaHyperParams as HyperParams, LlamaModel as Model,
+    LoadProgress, OutputToken,
```

Comment:
We should just do the renaming we discussed here :+1:

Reply:
Sounds good. Should I do `Vocab` -> `Vocabulary` and `Params` -> `Parameters`?

Reply:
Yup, sounds good